prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -248,7 +255,13 @@ export class CareSettingComponent extends BaseEnrolmentProfilePage implements On
         return clear(form.get('communityPharmacySites') as FormArray);
       }
       case CareSettingEnum.HEALTH_AUTHORITY: {
-        return clear(form.get('healthAuthoritySites') as FormArray);
+        const healthAuthoritySites = form.get('healthAuthoritySites') as FormGroup;
+        Object.keys(healthAuthoritySites.controls).forEach(healthAuthorityCode => {
+          const sitesOfHealthAuthority = healthAuthoritySites.get(String(healthAuthorityCode)) as FormArray;
```

Comment:
In the rest of the app using ES6 syntax we're converting to a string using ticks -  `${healthAuthorityCode}`

Reply:
Addressed